Lesson19/gozde/recursion.js

```diff
@@ -0,0 +1,210 @@
+// See completed in the lesson examples in the lesson files folder: https://github.com/frontend-from-0/group091123-files
+
+// 4. Write a recursive function to remove all occurrences of a specified character from a string.
+const sentenceEx5 = 'Hello, how are you?';
+const charToRemove = 'o';
+
+function removeCharRecursive(sentence, charToRemove) {
+    if (sentence.length === 0) {
+        return '';
+    }
+    
+    if (sentence[0] === charToRemove) {
+        return removeCharRecursive(sentence.slice(1), charToRemove);
+    } else {
+        return sentence[0] + removeCharRecursive(sentence.slice(1), charToRemove);
+    }
+}
+
+const sentenceEx5 = 'Hello, how are you?';
```

Comment:
This constant is declared above already, it's not allowed to create constants with the same name in the same scope. 
Solution: remove this line. 

There are a few more errors like this in this file. Run the file locally and you'll see where the errors are.